keyboards/ergodash/rev1/rev1.h

```diff
@@ -0,0 +1,68 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "ergodash.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+//void promicro_bootloader_jmp(bool program);
+
+#ifndef FLIP_HALF
+// Standard Keymap
+// (TRRS jack on the left half is to the right, TRRS jack on the right half is to the left)
+#define KEYMAP( \
```

Comment:
If you could use `LAYOUT_ortho_5x14` and remove the define at the bottom?  

We're moving away from "KEYMAP" in general, to the more hardware agnostic "layout".